.github/workflows/build_packages.yaml

```diff
@@ -130,7 +130,9 @@ jobs:
         fi
         
         if [ "${{ matrix.build_type }}" == "-msquic" ] || [ "${{ matrix.build_type }}" == "-full" ]; then
-          make_flags+=" -DNNG_ENABLE_QUIC=ON" 
+          if [ "$arch" != "mips" ] && [ "$arch" != "riscv64" ]; then
```

Comment:
Maybe we add new item in `strategy.matrix.exclude` is better

Reply:
Yeah, We did